cypress/integration/api/paste-assistant-thread.spec.ts

```diff
@@ -1,4 +1,4 @@
-context("POST /api/paste-assistant-thread", () => {
+context("/api/paste-assistant-thread", () => {
```

Comment:
good catch, I was rewriting these endpoints

Reply:
oh, wait, no.  this is just a label, and because I have more request types than just GET of got rid of it. It's nothing to do with the actual call or test.